content/en/docs/concepts/policy/resource-quotas.md

```diff
@@ -569,6 +569,15 @@ For example:
 
 See [LimitedResources](https://github.com/kubernetes/kubernetes/pull/36765) and [Quota support for priority class design doc](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/scheduling/pod-priority-resourcequota.md) for more information.
 
+## ResourceQuota vs LimitRange

```

Comment:
Please remove the unrelated part from this PR. I see you have another PR to add this part.

Reply:
I don't know why it got mixed up.
I removed the unrelated PR.